src/Core/src/Platform/iOS/MauiButton.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using UIKit;
+
+namespace Microsoft.Maui.Platform
+{
+	public class MauiButton : UIButton
```

Comment:
Unfortunately, it looks like this breaks Button and makes them all get set to custom